### PR TITLE
add contribute guide

### PR DESCRIPTION
**Pull request by vnechaev**
This is only the draft of contributing guide. Let's use it for start of discussion. Please, give me feedback on points in contributing guide

### CODE REVIEW

File: contribute.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+Fork the repository to your GitHub Account.
+Clone forked repo to your local machine
+Create your feature branch
+Make code/documantation changes
+Create pull request
+
+Before sending your pull requests, make sure you followed this list.
+
+Read contributing guidelines.
+Read Code of Conduct.
```
Comment (LuchnikovI) [inline contribute.md:31]: Where should the Code of Conduct be placed? Should it be a separate file, or can one add it just here?

File: contribute.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+Ensure you have signed the Contributor License Agreement (CLA).
```
Comment (LuchnikovI) [inline contribute.md:32]: Again, due to a lack of my education in the programming of open source software, what is Contributor License Agreement, and where should it be placed?

Comment (vnechaev) [reply contribute.md:32]: Contributor License Agreement(CLA) is optional.It is a document resplendent with intellectual
property implications and therefore, a controversial matter for some free and open source software practitioners.
If you have no special requirements to contributors in area of intellectual property implications, you should not create CLA.
CLA has administrative overhead and slowdown in contributions.
Can be placed as text file anywhere in project. You can guarantee agreement by git check - contributor signing their contribution(commit) using the -s or --signoff flags of the git version control system.

File: contribute.md (hunk 1 above)
Comment (vnechaev) [reply contribute.md:31]: It is usual separate file, and threre is a link to code of conduct in readmi

Comment (vnechaev) [reply contribute.md:31]: Code of Conduct is optional, it is actual for project with contributors community

**Pull request merged**